<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1118</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>256</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Sans Serif&quot;;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>261</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>221</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="text">
        <string>Enter new name here..</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>17</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="rename_button">
         <property name="text">
          <string>Rename</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>17</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="rename_button_2">
         <property name="text">
          <string>Done</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>textBrowser</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1118</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuVersion">
    <property name="title">
     <string>Version</string>
    </property>
    <addaction name="actionCurrent_Version"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionOpen_Help_Page"/>
   </widget>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>Exit</string>
    </property>
    <addaction name="actionExit_Hi_OS"/>
   </widget>
   <widget class="QMenu" name="menuDate_Time">
    <property name="title">
     <string>Date &amp; Time</string>
    </property>
    <addaction name="actionDate"/>
    <addaction name="actionTime"/>
   </widget>
   <widget class="QMenu" name="menuDirectory">
    <property name="title">
     <string>Directory</string>
    </property>
    <addaction name="actionShow_Directory"/>
   </widget>
   <widget class="QMenu" name="menuRename">
    <property name="title">
     <string>Rename</string>
    </property>
    <addaction name="actionRename_Stuff"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuVersion"/>
   <addaction name="menuDate_Time"/>
   <addaction name="menuHelp"/>
   <addaction name="menuRename"/>
   <addaction name="menuDirectory"/>
   <addaction name="menuExit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionGo_to_PCB_Screen"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCurrent_Version">
   <property name="text">
    <string>Current Version</string>
   </property>
  </action>
  <action name="actionExit_Hi_OS">
   <property name="text">
    <string>Exit Hi! OS</string>
   </property>
  </action>
  <action name="actionDate">
   <property name="text">
    <string>Date</string>
   </property>
  </action>
  <action name="actionTime">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show date and time (coming soon</string>
   </property>
  </action>
  <action name="actionShow_Directory">
   <property name="text">
    <string>Show Directory</string>
   </property>
  </action>
  <action name="actionOpen_Help_Page">
   <property name="text">
    <string>Open Help Page</string>
   </property>
  </action>
  <action name="actionRename_Stuff">
   <property name="text">
    <string>Rename Stuff</string>
   </property>
  </action>
  <action name="actionGo_to_PCB_Screen">
   <property name="text">
    <string>Go to PCB Screen</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
